compactHeadersApi.js

```diff
@@ -102,7 +102,7 @@ function install(window) {
   compactHeadersSeparator4.id = "compactHeadersSeparator4";
 
   let expandedfromRow = document.getElementById("expandedfromRow");
-  expandedfromRow.setAttribute("style", "align-items: center; margin-block: -1em; padding-block: 1em; margin-inline: -2px auto; overflow: hidden; min-width: min-content;");
+  expandedfromRow.setAttribute("style", "align-items: center; margin-block: -1em; padding-block: 1em; margin-inline: -2px auto; overflow: hidden; min-width: max-content;");
```

Comment:
By using `min-width: max-content`, if the Address line does not contain sufficient space, the `From` Header is guaranteed to always fully display:
![image](https://github.com/dillenger/compact-headers/assets/20227484/0ac3c83b-967b-4e90-bfbf-8ad9cbd58e44)